bin/postfix_init.sh

```diff
@@ -38,5 +39,35 @@ echo "Disable chroot for the smtp service"
 postconf -F smtp/inet/chroot=n
 postconf -F smtp/unix/chroot=n
 
+if [[ "${POSTFIX_TLS}" = "true" ]]; then
+    echo "Configuring TLS"
```

Comment:
Antonia has been bugging me lately about using single quotes for echo statements that require no interpolation.  This is a real bike shed discussion on style but it would be good to pick a direction going forward.

Reply:
I agree with using the correct quotes interpolating vs. not... I'll fix

Reply:
Hmm.. `shellcheck` should catch this I think